### PR TITLE
Prepare v8 release

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

**Comment by nondiremanuel** [comment]
Seems correct to me!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+
+#### Liana daemon / library
+
+A new field `last_poll_timestamp` was added to the wallet table in order to track the last time the wallet
+was updated by the poller.
+
+#### Liana GUI
+
+- The new Bitcoind 28.0 version is now downloaded by the GUI.
+- Bitbox02 devices can now be part of a Taproot descriptor set-up if their firmware version is superior or equal to 9.21.0.
```
Comment (jp1ac4) [inline CHANGELOG.md:28]: "firmware version is greater than or equal to"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+- The install process includes now example of setups with fixed templates and explanatory description to guide the user.
```
Comment (jp1ac4) [inline CHANGELOG.md:29]: "The install process now includes examples of setups with fixed templates and explanatory descriptions to guide the user."

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
 # Liana daemon and GUI release notes
 
+## 8.0
+
+This release introduces support for Bitbox02 device for a Taproot descriptor and
```
Comment (jp1ac4) [inline CHANGELOG.md:5]: perhaps "... support for Taproot descriptors on Bitbox02 devices..."

**Review by jp1ac4 — COMMENTED** [review]
A few suggested wording changes but otherwise all good.

**Review by jp1ac4 — APPROVED** [review]
ACK https://github.com/wizardsardine/liana/commit/9c30aad9e2f245e587eb5bcd689383e4215b1ed6.

**Pull request merged**